Decred Wallet/Custom Views/FloatingLabelTextField.swift

```diff
@@ -0,0 +1,119 @@
+//
+//  FloatingLabelTextField.swift
+//  Decred Wallet
+//
+//  Created by Sprinthub on 15/08/2019.
+//  Copyright © 2019 Decred. All rights reserved.
+//
+
+import UIKit
+
+class FloatingLabelTextField: UITextField {
+    
+    
+    var border = CALayer()
+    var floatingLabel: UILabel!
+    var placeHolderText: String?
+    
+//    override var isSecureTextEntry: Bool = false
+    
+    @IBInspectable
+    var floatingLabelColor: UIColor = UIColor.appColors.decredBlue {
+        didSet {
+            self.floatingLabel.textColor = floatingLabelColor
+        }
+    }
+    
+    
+    
+    @IBInspectable
+    var floatingLabelFont: UIFont = UIFont.systemFont(ofSize: 14) {
+    
+        didSet {
+            self.floatingLabel.font = floatingLabelFont
+        }
+    }
+        
+    var floatingLabelHeight: CGFloat? = 14
+    
+    let maxLabelSize: CGSize = CGSize(width: 1024, height: 1024)
+    
+    
+    var button = UIButton(type: .custom)
```

Comment:
These declarations at lines 37, 39 and 42 can move to the top, without empty line separating them unless they're unrelated.

Reply:
resolved in e7f7df7